CHANGELOG.md

```diff
@@ -7,7 +7,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## 0.4.0 - 2020-04-23
```

Comment:
I was going to say this should be a link to the comparison delta, but I see we missed setting that anywhere else in the changelog yet. 🤔 I'll fix as a separate commit.

Reply:
https://github.com/amperity/greenlight/commit/71ff19997dbbb3796470a5068b61132920c988ab